frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -233,8 +233,8 @@ function ReaderHighlight:init()
 end
 
 function ReaderHighlight:setupTouchZones()
-    -- deligate gesture listener to readerui
-    self.ges_events = {}
+    -- delegate gesture listener to readerui
+    self.ges_events = nil
```

Comment:
@NiLuJe 
Regression

The 4 places you changed `self.ges_events` from `{}` to `nil` do not let me do any gestures in the reader (even swipe to open the menu)

`./luajit: frontend/ui/widget/container/inputcontainer.lua:257: bad argument #1 to 'pairs' (table expected, got nil)`